src/components/Tablecell.tsx

```diff
@@ -0,0 +1,140 @@
+import { Tooltip } from 'react-tooltip';
+import { Block, ISociety } from "../data/societies";
+import { GridItem } from "./Grid";
+import Modal from 'react-modal'
+import { useState } from 'react';
+import facebook from '../assets/socials_facebook.png';
+
+export type TableCellProps = { society: ISociety, index: number, type: "society" } | { index: number, type: "undiscovered", block: Block } | { type: "empty" | "inner-transition" | "the-unknown-soc" };
+
+
+function getIUPACTemporaryNameAndSymbol(index: number): { name: string, symbol: string } {
+	const NUMERICAL_ROOTS = ["nil", "un", "bi", "tri", "quad", "pent", "hex", "sept", "oct", "en"];
+	const index_digits = index.toString().split('').map((digit_str) => parseInt(digit_str));
+
+	let symbol = index_digits.map((digit) => NUMERICAL_ROOTS[digit][0]).join('');
+	symbol = symbol[0].toUpperCase() + symbol.slice(1);
+
+	let name = index_digits.map((digit) => NUMERICAL_ROOTS[digit]).join('');
+	name.replace('nnn', 'nn');
+	name.replace('iii', 'ii');
+	name = name[0].toUpperCase() + name.slice(1) + "ium";
```

Comment:
Can you move this statement up before the two `.replace()` statements? This was a bug I just realized :stuck_out_tongue: